airbyte-integrations/connectors/source-jira/source_jira/source.py

```diff
@@ -81,7 +81,7 @@ def get_authenticator(config: Mapping[str, Any]):
     def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) -> Tuple[bool, Optional[Any]]:
         config = self._validate_and_transform(config)
         authenticator = self.get_authenticator(config)
-        kwargs = {"authenticator": authenticator, "domain": config["domain"], "projects": config["projects"]}
+        kwargs = {"authenticator": authenticator, "domain": config["domain"], "projects": config.get("projects", [])}
```

Comment:
can you please move this to `_validate_and_transform` function and make such transformation:

```
config["projects"] = config.get("projects", [])
```

?

Reply:
Done :+1: